app/src/main/java/com/yeonae/chamelezone/view/place/presenter/PlaceReviewPresenter.kt

```diff
@@ -14,7 +15,11 @@ class PlaceReviewPresenter(private val reviewRepository: ReviewRepository,
     override fun placeDetailReview(placeNumber: Int) {
         reviewRepository.getReviewList(placeNumber, object : ReviewCallBack<List<ReviewResponse>> {
             override fun onSuccess(response: List<ReviewResponse>) {
-                placeReviewView.showPlaceReview(response)
+                val reviewItem = arrayListOf<ReviewItem>()
+                response.forEachIndexed { index, _ ->
+                    reviewItem.add(response[index].toReviewItem(response[index]))
+                }
```

Comment:
```suggestion
                response.forEach {
                    reviewItem.add(it.toReviewItem())
                }
```
toReviewItem을 하는데 자기자신을 파라미터로 넣을 필요가 있나요?
자기자신이라 이미 알고있으므로 불필요한 부분 같구요.
index를 참조해서 사용할 필요 없는 부분같습니다.

Reply:
수정했습니다
d19bab2

Reply:
```
toReviewItem을 하는데 자기자신을 파라미터로 넣을 필요가 있나요?
자기자신이라 이미 알고있으므로 불필요한 부분 같구요.
```
에 대해 반영이 없는 것 같아요